src/components/InventoryGroups/Modals/ConfirmSystemsAddModal.js

```diff
@@ -1,7 +1,7 @@
 import { FormSpy, useFormApi } from '@data-driven-forms/react-form-renderer';
 import { Button, Flex, Icon } from '@patternfly/react-core';
 import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
-import warningColor from '@patternfly/react-tokens/dist/esm/global_warning_color_100';
+import { warningColor } from '@patternfly/react-tokens';
```

Comment:
This should have been changed too, like the ones above. it is now raising an error when building: 
![Screenshot 2024-06-26 at 13 42 54](https://github.com/RedHatInsights/insights-inventory-frontend/assets/7757/c51c720b-a018-4291-b48a-351aac226489)